fhttp/httprunner.go

```diff
@@ -193,14 +206,27 @@ func RunHTTPTest(o *HTTPRunnerOptions) (*HTTPRunnerResults, error) {
 		total.sizes.Transfer(httpstate[i].sizes)
 		total.headerSizes.Transfer(httpstate[i].headerSizes)
 	}
+
+	// Sort the ip address form largest to smallest based on its usage count
```

Comment:
I think this is more complicated that it needs to be, see how `keys` and `total.RetCodes[k]` is handled (not the best name `keys` but still same principle)

Reply:
though to sort by value vs by key is indeed a bit more involved but not much:
```golang
    sort.Slice(keys, func(i, j int) bool { return counts[keys[i]] > counts[keys[j]] })
```

Reply:
updated